samples/05-file-transfer-cloud/transfer-file/src/main/java/org/eclipse/dataspaceconnector/extensions/transfer/CloudTransferExtension.java

```diff
@@ -30,7 +30,8 @@ public String name() {
     public void initialize(ServiceExtensionContext context) {
         var dataFlowMgr = context.getService(DataFlowManager.class);
         var dataAddressResolver = context.getService(DataAddressResolver.class);
-        var blobStoreApi = context.getService(BlobStoreApi.class);
```

Comment:
Why not using the BlobStoreApi instance already registered?

Reply:
The extension, that implements the BlobStoreApiImpl class, is an library and does not register it in the service context. Maybe this was different in the past.

Reply:
Yes, sorry, it gets registered in the `blob-provision` module, that's not included here.